locale/en/messages.json

```diff
@@ -165,8 +165,55 @@
   "blm.thunder.total-clip": "Total Clip",
   "blm.triplecast.suggestions.overwrote-triplecasts.content": "You lost at least one instant cast spell by using <0/> while the status was already active.",
   "blm.triplecast.suggestions.overwrote-triplecasts.why": "You overwrote <0/> .",
-  "blu.about.description": "This is a limited analyzer for a limited job. It's not ready yet, but please look forward to it.\n\nThis analyzer will focus on DoTs, uptime, and proper usage of [~action/BRISTLE],[~action/MOON_FLUTE], and [~action/SURPANAKHA].\n",
-  "blu.about.description.warning": "This module is under development and not yet ready for the public.",
+  "blu.about.description": "BLU has wildly different play styles depending on your mimickry.\n\nThis page will primarily focus on the DPS aspect of the job, so do take the advice with a grain of salt if you are healing or tanking.\n\nWorth also keeping in mind the usual adage that BLU content is won and lost on mechanics, not DPS.  Following the suggestions here to optimize your performance will always be welcome, but it should not detract from mechanics; greeding Phantom Flurry ticks at the expense of your team mate's sanity is not recommended.\n\nIf you want further context for the suggestions given here, the [Blue Academy Discord](https://discord.gg/blueacademy) is the primary source for Blue Mage information.",
```

Comment:
Mimicry, mentioned in another comment as well.

Reply:
Patched in ff3f05c59604bb1ac24a4e8aa90d275f2920cf0c